db/migrate/20170103144759_encrypt_user_credentials.rb

```diff
@@ -0,0 +1,6 @@
+class EncryptUserCredentials < ActiveRecord::Migration[5.0]
+  def change
+      rename_column :user_credentials, :credential_value, :encrypted_credential_value
```

Comment:
I think we're going to need to leave this alone so as to not break people's existing installations. Maybe we could add an `encrypted` boolean flag and treat the `credential_value` as encrypted when that's true?  In terms of migrating, I guess we'd need to have code that runs in an initializer that migrates the DB to be encrypted if `ENV['APP_ENCRYPTION_PASSPHRASE']` is set? It'd be weird because we can't use a normal migration because `ENV['APP_ENCRYPTION_PASSPHRASE']` might not be set yet when someone updates to the newer version of Huginn, and then the traditional migration would run and they wouldn't be able to move to encryption anymore.